src/app/systelab-components/chip-button/chip-button.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { MessagePopupService } from '../modal';
+import { I18nService } from 'systelab-translate/lib/i18n.service';
+
+export class ChipButtonItem {
+	constructor(public id: number, public name: string, public isChecked = false) {
+	}
+}
+
+@Component({
+	selector:    'systelab-chip-button',
+	templateUrl: 'chip-button.component.html',
+})
+export class ChipButtonComponent {
+
+	@Output() public changeButton = new EventEmitter();
+	@Output() public selectedButton = new EventEmitter();
```

Comment:
Rename to selectButton because it is an action

Reply:
Done